cmd/cluster/context.go

```diff
@@ -301,8 +306,110 @@ func (o *contextOptions) printPDAlerts() error {
 		fmt.Println("error while flushing table: ", err.Error())
 		return err
 	}
+	return nil
+}
+
+func printHistoricalPDAlertSummary(pdClient *pd.Client, ctx context.Context, serviceID string) error {
+
+	fmt.Println()
+	fmt.Println("============================================================")
+	fmt.Println("Historical Pagerduty Alert Summary")
+	fmt.Println("============================================================")
+	fmt.Printf("Link to PD Service: https://redhat.pagerduty.com/service-directory/%s\n", serviceID)
+
+	var currentOffset uint
+	var limit uint = 100
+	var incidents []pd.Incident
+	print("\nPulling historical pd data")
+	for currentOffset = 0; true; currentOffset += limit {
+		print(".")
+		// pd defaults pulling the past month of data, which is enough for us to work with
+		liResponse, err := pdClient.ListIncidentsWithContext(
+			ctx,
+			pd.ListIncidentsOptions{
+				ServiceIDs: []string{serviceID},
+				Statuses:   []string{"resolved", "triggered", "acknowledged"},
+				Offset:     currentOffset,
+				Limit:      limit,
+				SortBy:     "created_at:desc",
+			},
+		)
+
+		if err != nil {
+			fmt.Printf("Failed to ListIncidentsWithContext %q\n", err)
```

Comment:
This will cause a double print of the error, as line 408 would print this error as well.

Reply:
Removed it :)